lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -145,7 +145,7 @@ def _handle_dropped_request(self, result, d, function_name):
 
     def render(self, request):
         time_in = utils.now()
-        assert self._check_headers(request), InvalidHeaderError
+        # assert self._check_headers(request), InvalidHeaderError
```

Comment:
jack - you probably don't like that I did this, but electron is now essentially a separate process the queries the daemon via rpc and the checks in check_headers didn't seem that useful.  Open to being convinced otherwise.